package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/distroless/base
+FROM busybox
```

Comment:
If we have to change OS anyway, move to SLE-BCI: https://github.com/rancher/eks-operator/blob/master/package/Dockerfile#L1

Reply:
will do

Reply:
make sure you do a base test with this to, making sure the hook fires when config map changes